bin/commands/deposit.js

```diff
@@ -242,7 +242,7 @@ const commandParsers = {
     } else {
       return async tbtc => {
         const deposit = await tbtc.Deposit.withAddress(depositAddress)
-        return redeemDeposit(tbtc, deposit, "")
+        return redeemDeposit(tbtc, deposit, redemptionBitcoinAddress)
```

Comment:
To understand what this was doing, I dug into the source for `redeemDeposit`.

```
      let redemption
      if (typeof redemptionInfo == "string") {
        redemption = await deposit.requestRedemption(redemptionInfo)
      } else {
```

Did this work at all before?